### DISCUSSION TITLE
routing, netlify

### DISCUSSION THREAD

**Discussion by thernode** [vercel/next.js#11956] [Help]
> example used
https://github.com/zeit/next.js/tree/canary/examples/dynamic-routing

| link| check |
| --- | --- |
| https://dynamic-routing-8pu7kv9v9.now.sh/post/first | ✔️|
| https://dynamic-routing-8pu7kv9v9.netlify.app/post/first | ❌|

How can I edit the router problem for the netlify site?

**Comment by timneutkens** [comment]
`next export` does not support all flavors of dynamic routes out of the box as it requires proxy-level routing. On ZEIT this is automatically configured as Next.js is integrated into the platform. 

`next export` has a few other limitations, for example API routes don't work: https://nextjs.org/docs/api-routes/introduction

If you want to statically render all path segments you can use https://nextjs.org/docs/basic-features/data-fetching#getstaticpaths-static-generation which works with `next export`.
